ui/app/templates/permissions_modal.html

```diff
@@ -10,7 +10,7 @@ <h3 class="modal-title">{{ user.username }}</h3>
           <a href="#" ng-click="currentItemTab = 1;">Permissions</a>
         </li>
         <li ng-class="{active: currentItemTab == 2}">
-          <a href="#" ng-click="currentItemTab = 2;">Roles</a>
+          <a href="#" ng-click="currentItemTab = 2;" ng-hide="showRole" ng-show='user.permissions.length > 0'>Roles</a>
```

Comment:
Maybe the entire `li` item should be conditioned and not the inner link.. i.e put these conditions at Line 12 above..